.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+<!--
+* New contributors are highly encouraged to read our
+  [CONTRIBUTING](/CONTRIBUTING.md) documentation.
+* Commit and PR titles should be prefixed with the general area of the pull request's change.
+
+-->
+### What does this PR do?
+
+<!--
+* A brief description of the change being made with this pull request.
+* If the description here cannot be expressed in a succinct form, consider
+  opening multiple pull requests instead of a single one.
+-->
+
+### Motivation
+
+<!--
+* What inspired you to submit this pull request?
+* Link any related GitHub issues or PRs here.
+* If this resolves a GitHub issue, include "Fixes #XXXX" to link the issue and auto-close it on merge.
+-->
+
+### Describe how to test/QA your changes
+
+<!--
+* Write here in detail or link to detailed instructions on how this change can
+  be tested/QAd/validated, including any environment setup.
+* Ideally your changes have automated unit and/or integration tests which are run in CI.
+-->
+
+### Reviewer's Checklist
+<!--
+* Authors can use this list as a reference to ensure that there are no problems
+  during the review but the signing off is to be done by the reviewer(s).
+-->
+
+- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
```

Comment:
This one would confuse me a contributor. How would I ever know the appropriate milestone to select? Especially if I'm the kind of person this template is meant to guide?

Reply:
Maybe we can ask to use the current milestone as a default? a.k.a the closest one to the current version. I'm guessing the majority of PRs belonging to `Triage` would end up being moved this way anyway.

Reply:
We can certainly clarify (although I think external contributors may lack permissions to set it?) However this section is part of the reviewers checklist so that reviewers can use it to track their review, so I wasn't necessarily expecting the PR author to take any action here